open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb

```diff
@@ -49,7 +49,13 @@
     "\n",
```

Comment:
<p>There are too many pictures attached, add notes as appropriate, or put the pictures in the right place for instruction</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>